security/advancedtls/examples/credential_reloading_from_files/client/main.go

```diff
@@ -98,10 +100,12 @@ func main() {
 	// the bash script. We don't cancel the context nor call conn.Close() here,
 	// since the bash script is expeceted to close the client goroutine.
 	for {
-		_, err = client.SayHello(context.Background(), &pb.HelloRequest{Name: "gRPC"}, grpc.WaitForReady(true))
+		ctx, cancel := context.WithDeadline(context.Background(), time.Now().Add(defaultDeadline))
```

Comment:
```go
ctx, cancel := context.WithTimeout(context.Background(), defaultDeadline)
```

To be pedantic: make it `defaultTimeout`; a "deadline" is a `time.Time`.

Reply:
Done. Thanks again for the review!